app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1227,36 +1230,36 @@ public void updateVisibleWidgets() {
     }
 
     @Override
-    public void startWidgetResize(final Widget aWidget, float aMaxWidth, float aMaxHeight, float minWidth, float minHeight) {
+    public void startWidgetResize(@NonNull final Widget aWidget, float aMaxWidth, float aMaxHeight, float minWidth, float minHeight) {
```

Comment:
Should never be null but could potentially be null when called from:
https://github.com/MozillaReality/FirefoxReality/blob/2516a84dcc5b425a9178dc3f441938ce57e8d2b6/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java#L985

Reply:
We need need to fix the base class the.